core/storage/suggestion/gae_models.py

```diff
@@ -631,8 +643,8 @@ def get_export_policy(cls):
             'status': base_models.EXPORT_POLICY.EXPORTED,
             'content': base_models.EXPORT_POLICY.EXPORTED,
             'filename': base_models.EXPORT_POLICY.EXPORTED,
-            'author_id': base_models.EXPORT_POLICY.EXPORTED,
-            'final_reviewer_id': base_models.EXPORT_POLICY.EXPORTED,
+            'author_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Same note as above, maybe add comments above this line and the one below to explain why this is considered "not applicable".

Reply:
done